lib/core.ps1

```diff
@@ -1415,6 +1423,8 @@ if ($pathExpected) {
 }
 $scoopConfig = load_cfg $configFile
 
+$scoop_path_env = get_config PATH_ENV 'PATH'
```

Comment:
Recommend using a config option to check if use isolated path.

Reply:
I assumed we would want the default behavior to revert to the current behavior of just adding the paths to `PATH`. So we should have two different config options? A bool to enable custom paths and one to contain the path itself? I feel it makes more sense to just have a single config value that contains the Environment variable scoop should write to. That way we don't need extra checks and the user is always aware which environment variable is being written to.